frame/babe/src/lib.rs

```diff
@@ -190,12 +190,12 @@ pub mod pallet {
 		InvalidConfiguration,
 	}
 
-	/// Current epoch index.
+	/// Current session index.
 	#[pallet::storage]
-	#[pallet::getter(fn epoch_index)]
+	#[pallet::getter(fn session_index)]
```

Comment:
For now, for the storage, we only renamed the `getter` methods.
**NO RENAME** for the storage identifiers has been performed in this PR.

Renaming storage identifier changes the storage prefix, thus it requires a migration.

Not renamed storage identifiers: `EpochIndex`, `PendingEpochChangeConfig`, `EpochStart`, `EpochConfig`, `NextEpochConfig`.

Identifiers renaming with a migration will be performed in a follow-up PR to better isolate the change.

M